### PR TITLE
Working home but the UI changes are not visible

### PR DESCRIPTION
**Pull request by arpit1912**
@sam3926  just take a look and let me know what changes I should make.

### CODE REVIEW

**Comment by sam3926** [comment]
@arpit1912 home left side pane collapsible doesn't seem to be working. Also on clicking your profile in left home pane, the profile page should open. In posts, the bottom buttons only work if you click directly on button and not nearby even though mouse pointer turns clickable. Also like button should change on clicking (@dinkar64  can help you with last two things)

**Comment by sam3926** [comment]
also @arpit1912 edit post template to keep only date at right side (for now there are two components there with a break statement even though only one date needs to be displayed). Posts in profile section also needs to be connected and edited to be same template

File: frontend/src/components/HomeCenter/index.js [hunk 1]
```diff
@@ -97,11 +103,11 @@ class HomeCenter extends Component {
         const postList = posts.length? (
             posts.map(post =>{
                 return(
-                    <Card title={post.title} extra={<div>{post.user_name}<br></br>{post.date}</div>} style={{ width: 700 , margin:"8px"}} 
+                    <Card title={post.author} extra={<p>{post.DateTime}</p>} style={{ width: 700 , margin:"8px"}}
```
Comment (sam3926) [inline frontend/src/components/HomeCenter/index.js]: @arpit1912  Remove paragraph tag here

**Review by sam3926 — CHANGES_REQUESTED** [review]

**Comment by arpit1912** [comment]
@sam3926 I have made the changes in the home pages check it out once.

**Comment by sam3926** [comment]
@arpit posts look fine now, just gotta settle the dates issue
Home page work left:
1. link left pane profile to self profile page
2. make left pane collapsible working

Also, is there any way to increase like/share button clickable area even more?

**Comment by arpit1912** [comment]
The date issue is resolved..try to create new post, the date is in the required format.. if not coming maybe I will push it into the next pr as it's there in my local branch

**Comment by sam3926** [comment]
Cool
@arpit1912 i've edited the profile post layout to match home one, link it to the backend

**Comment by arpit1912** [comment]
@sam3926  @dinkar64  @krish7777  checkout the working donation section of the discover page meanwhile I will request you guys to create some post as I need it for checking of filters do check the update in the expiry date picker as well :smile:

**Comment by arpit1912** [comment]
> Cool
> @arpit1912 i've edited the profile post layout to match home one, link it to the backend

@sam3926 Sure! I thought you merged this branch so committed here.. anyways try to close it asap...Will do your work In the next PR

**Comment by arpit1912** [comment]
@dinkar64  now the interest and reject buttons on discover and profile page are working as well as changing the database in real time

**Pull request merged**